HotelBooking/HotelBooking.App/Operations/EmailSender.cs

```diff
@@ -0,0 +1,26 @@
+﻿using HotelBooking.Domain.Models;
+using HotelBooking.App.Operations.ExternalSystems;
+using System;
+using HotelBooking.Domain.Enums;
+
+namespace HotelBooking.App.Operations
+{
+    public class EmailSender : OperationBase
+    {
+        private IExternalEmailSystem _emailSender;
+
+        public EmailSender(IExternalEmailSystem emailSender)

```

Comment:
To dotyczy chyba wszystkich twochich operacji. Nie wiem jakie plan mialas w glowie jesli deleguje cala funksjonalnosc do wspolpraownika to poza IsRequired nie widze za co ta klasa jest odpowiedzialna. Innymi slowy dla mnie to chyba troche za duzo tutaj delegacji ale to sprawa mocno koncpcyjna - do dyskusji